include/interface/blas2_interface.h

```diff
@@ -213,6 +213,43 @@ typename executor_t::policy_t::event_t _syr2(
     container_2_t _mA,  // (_lda, _N) The output matrix
     index_t _lda        // >max(1, _N) The first dimension of _mA
 );
+
+/*!
+ @brief Generalised matrix vector product with band matrices.
+
+ Generalised matrix vector product with a band matrix, i.e. computing the
+ mathematical operation:
+
+ y = alpha*A*x + beta*y
+
+ See the netlib blas interface documentation for more details of the high level
+ interface: https://netlib.org/lapack/explore-html/d6/d46/sgbmv_8f.html
+ */
+template <typename executor_t, typename index_t, typename element_t,
+          typename container_0_t, typename container_1_t, typename increment_t,
+          typename container_2_t>
+typename executor_t::policy_t::event_t _gbmv(
+    executor_t& ex,     // executor_t (sycl, parallel, serial, etc)
```

Comment:
I think it would be preferable to have the documentation for the parameters in a format compatible with doxygen (i.e. inside /** */ or /*! */) . It would also match what is being done on other sycl-blas api's. Same comment would apply to other declarations of _gbmv

Reply:
After looking at the code, it seems that level 2 apis are all using this style. While level 1 are using doxygen. I still prefer the standardized style but would understand if you want to keep this one since the whole blas2_interface.h is using it.

Reply:
I have no preference. I followed the style of `gemv`.

Let me know which is the one that needs to be used.

Reply:
The last commit applies the doxygen format.